main/src/main/scala/sbt/Keys.scala

```diff
@@ -133,6 +149,8 @@ object Keys {
   val managedSources = taskKey[Seq[File]]("Sources generated by the build.").withRank(BTask)
   val sources = taskKey[Seq[File]]("All sources, both managed and unmanaged.").withRank(BTask)
   val sourcesInBase = settingKey[Boolean]("If true, sources from the project's base directory are included as main sources.")
+  val inputs = settingKey[Seq[Glob]]("The file globs that are used by a task. This setting will generally be scoped per task. It will also be used to determine the sources to watch during continuous execution.")
+  val triggers = settingKey[Seq[Glob]]("Describes files that should trigger a new continuous build.")
```

Comment:
1. When would you need this, instead of `foo / inputs`?
2. `watchTriggers` might be a better name since the trigger might come up in different context like `def trigger = allRequirements`

Reply:
Yeah. That's a good point. I also am planning to move these keys into the plugin we discussed.